build-with-pip.file

```diff
@@ -43,6 +43,9 @@
 
 %define pip pip3&package_dependency=py3-pip
 Requires: python3 py3-pip py3-setuptools
+%if "%{n}" != "py3-wheel"
```

Comment:
hi @smuzaffar - I was looking at python packages too.. Getting errors related to wheels  is likely a sign that you need to update setup tools

Reply:
I tried building latest setuptools but was not able to build it itself. It was complainging about missing build/script-39 directory.